scripts/secnetperf.ps1

```diff
@@ -137,6 +137,44 @@ Copy-Item -ToSession $Session ./artifacts -Destination "$RemoteDir/artifacts" -R
 Copy-Item -ToSession $Session ./scripts -Destination "$RemoteDir/scripts" -Recurse
 Copy-Item -ToSession $Session ./src/manifest/MsQuic.wprp -Destination "$RemoteDir/scripts"
 
+# Create the logs directories on both machines.
+New-Item -ItemType Directory -Path ./artifacts/logs | Out-Null
+Invoke-Command -Session $Session -ScriptBlock {
+    New-Item -ItemType Directory -Path $Using:RemoteDir/artifacts/logs | Out-Null
+}
+
+# Collect some info about machine state.
+if ($isWindows) {
+    $Arguments = "-SkipNetsh"
```

Comment:
I suspect we may want the `netsh` output.

Reply:
Let's start small.